charts/kubernetes/templates/playbooks/playbooks.yaml

```diff
@@ -165,11 +165,113 @@ spec:
           kubectl create rolebinding "playbook-$(.run.id)"
           --namespace=$(.config.name)
           --user=$(.params.user)
-          --role=$(.params.role)
+          --role=$(.params.role.name)
     - name: Clear Rolebinding
       if: "success()"
       delay: ".params.delay"
       exec:
         script: |
           kubectl delete rolebinding "playbook-$(.run.id)" --namespace=$(.config.name)
 {{- end}}
+
+{{- if and .Values.playbooks .Values.playbooks.requestNamespaceAccessGitops }}
+---
+apiVersion: mission-control.flanksource.com/v1
+kind: Playbook
+metadata:
+  name: request-namespace-access-gitops
+spec:
+  configs:
+    - types:
+        - Kubernetes::Namespace
+  title: Request Namespace Access (GitOps)
+  category: Kubernetes
+  description: Adds a Rolebinding to the given role in the selected Namespace
+  parameters:
+    - name: repo
+      label: Repository URL
+    - name: base_branch
+      label: Base branch
+    - name: author_name
+      label: Commit author
+    - name: author_email
+      label: Commit author Email
+    - name: connection
```

Comment:
we should probably add a new parameter type for connection so it's a dropdown
https://docs.flanksource.com/reference/playbooks/parameters#types